recipes/psycopg2/build.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+export LDFLAGS="${LDFLAGS} -L$PREFIX/lib -lssl"
+
+$PYTHON setup.py install
+
+if [[ `uname` == 'Darwin' ]]; then
+     PG_LIB=$(pg_config --libdir)
+     for LIBRARY in `find ${SP_DIR}/${PKG_NAME} -name "*.so"`;
+     do
+        install_name_tool -change libssl.1.0.0.dylib @rpath/libssl.1.0.0.dylib $LIBRARY
+        install_name_tool -change libcrypto.1.0.0.dylib @rpath/libcrypto.1.0.0.dylib $LIBRARY
```

Comment:
If Continuum would build openssl with conda-build this wouldn't be needed (c.f. https://github.com/ContinuumIO/anaconda-issues/issues/498).